contribs/gnodev/internal/mock/server_emitter.go

```diff
@@ -7,6 +7,12 @@ import (
 	"github.com/gnolang/gno/contribs/gnodev/pkg/events"
 )
 
+// Define empty event for NextEvent empty queue
+var (
+	eventNull = events.Custom("NULL")
```

Comment:
Why do we need this?

Reply:
It's much easier to assume `NextEvent` returns a non-nil so you can directly use `require.Equal(t, "MyExpectedEvent", m.NextEvent().Type())` without having to nil-check the NextEvent result